src/styles/isomer-cms/pages/Editor.module.scss

```diff
@@ -47,42 +47,42 @@
   h1 {
     font-size: 3.375rem;
     line-height: 4.5rem;
-    font-weight: 400;
+    font-weight: 400 !important;
```

Comment:
Hmm these !importants are because of the values in `base.scss` right? I think we can actually remove the h1-h6 styles there - I looked through the template code, and couldn't find reference to that particular weight, and it doesn't seem to be used outside the editor currently. 

Non-blocking comment - if this turns out to break stuff i'm fine with reverting to `!important`, I just think we should try to avoid this as far as possible

Reply:
Yes is good practice to avoid this. Can use specificity of CSS to help here?